Libraries/Components/ToolbarAndroid/ToolbarAndroid.android.js

```diff
@@ -126,6 +126,14 @@ var ToolbarAndroid = React.createClass({
      */
     titleColor: ReactPropTypes.string,
     /**
+     * Content inset for the toolbar starting edge
```

Comment:
Please add an explanation like you did on this PR. Reading this I don't know what it does.